include/nccl_ofi.h

```diff
@@ -195,6 +222,8 @@ typedef struct pending_reqs_q {
 typedef struct nccl_ofi_handle {
 	char ep_name[MAX_EP_ADDR];
 	uint64_t tag;
+	/* Save temporary communicator state when creating send communicator */
+	save_comm_state_t state;
```

Comment:
Does this work with older versions of NCCL or will we have breakage similar to `aws` branch?